frontend/common/utils/format.js

```diff
@@ -90,6 +90,10 @@ const Format = {
       : Format.camelCase(sn)
   },
 
+  userDisplayName(person) {
```

Comment:
What's the reasoning behind creating a new function here vs just using Format.fullName directly in the OrganisationSettingsPage.js file?

Reply:
Corrected.